### DISCUSSION TITLE
Replacing Docker Container Files

### DISCUSSION THREAD

**Discussion by K-Tech-Software** [FieldRobotEvent/Virtual_Field_Robot_Event#55] [Q&A] [answered]
We have Problems with the `put_files_in_my_container.sh` for replacing files which have the same name. We wanted to update our b_container with new files. It does not overwrite our files.

Kind regards
AgcHOH Robotics

**Comment by ThijsRuigrok** [comment, marked as answer]
`put_files_in_my_container.sh` should replace the files in the destination. Can you give a bit more information on what you did. Maybe you can post the line of core you ran, and show the terminal output/result. 

The `put_files_in_my_container.sh` is actually just a wrapper around the `docker cp` command. Alternatively you can use the `docker cp` command directly https://docs.docker.com/engine/reference/commandline/cp/

**Reply by K-Tech-Software** [reply]
Thanks for your reply. 
We found a solution for our problem. We wanted to update our existing workspace in the docker container with new code which we had tested in Ubuntu in a real machine.
Our workaround is to move the /catkin folder out of the docker container, and use the `put_files_in_my_container.sh`script to copy a newer version of the workspace into the docker container.
